test/truefi2/lines-of-credit/TrueCreditAgency.test.ts

```diff
@@ -966,33 +966,15 @@ describe('TrueCreditAgency', () => {
     })
 
     describe('reverts if borrower', () => {
-      it('pool is not supported', async () => {
```

Comment:
I removed 3 duplicate tests (merge conflicts?) that are all unnecesary now that we're iterating through supported pools ourselves.